src/platform_impl/windows/dark_mode.rs

```diff
@@ -0,0 +1,257 @@
+/// This is a simple implementation of support for Windows Dark Mode,
+/// which is inspired by the solution in https://github.com/ysc3839/win32-darkmode
+use std::ffi::OsStr;
+use std::os::windows::ffi::OsStrExt;
+
+use winapi::{
+    shared::{
+        basetsd::SIZE_T,
+        minwindef::{BOOL, DWORD, UINT, ULONG, WORD},
+        ntdef::{LPSTR, PVOID, WCHAR},
+        windef::HWND,
+    },
+    um::{libloaderapi, uxtheme, winuser},
+};
+
+lazy_static! {
+    static ref WIN10_BUILD_VERSION: Option<DWORD> = {
+        unsafe {
+            let module = libloaderapi::LoadLibraryExW(
+                widestring("ntdll.dll").as_ptr(),
+                std::ptr::null_mut(),
+                libloaderapi::LOAD_LIBRARY_SEARCH_SYSTEM32,
+            );
+
+            let handle = libloaderapi::GetProcAddress(
+                module,
+                "RtlGetVersion\0".as_ptr() as _,
+            );
+
+            if handle.is_null() {
+                None
+            } else {
+                // FIXME: RtlGetVersion is a documented windows API,
+                // should be part of winit!
+
+                #[allow(non_snake_case)]
+                #[repr(C)]
+                struct OSVERSIONINFOW {
+                    dwOSVersionInfoSize: ULONG,
+                    dwMajorVersion: ULONG,
+                    dwMinorVersion: ULONG,
+                    dwBuildNumber: ULONG,
+                    dwPlatformId: ULONG,
+                    szCSDVersion: [WCHAR; 128],
+                }
+
+                #[allow(non_snake_case)]
+                let RtlGetVersion: unsafe extern "system" fn (
+                    *mut OSVERSIONINFOW
+                ) = std::mem::transmute(handle);
+
+                let mut vi = OSVERSIONINFOW {
+                    dwOSVersionInfoSize: 0,
+                    dwMajorVersion: 0,
+                    dwMinorVersion: 0,
+                    dwBuildNumber: 0,
+                    dwPlatformId: 0,
+                    szCSDVersion: [0; 128],
+                };
+
+                RtlGetVersion(&mut vi as _);
+
+                if vi.dwMajorVersion == 10 && vi.dwMinorVersion == 0 {
+                    Some(vi.dwBuildNumber)
+                } else {
+                    None
+                }
+            }
+        }
+    };
+
+    static ref DARK_MODE_SUPPORTED: bool = {
+        // We won't try to do anything for windows versions < 17763
+        // (Windows 10 October 2018 update)
+        match *WIN10_BUILD_VERSION {
+            Some(v) => v > 18362,
```

Comment:
```suggestion
            Some(v) => >= 17763,
```

I think is what you meant?